docs/sections/user_guide/yaml/components/jedi.rst

```diff
@@ -0,0 +1,44 @@
+.. _jedi_yaml:
+
+jedi
+====
+
+Structured YAML to run JEDI is validated by JSON Schema and requires the ``jedi:`` block, described below. If ``jedi`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
+
+Here is a prototype UW YAML jedi: block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/jedi.yaml
+
+UW YAML for the ``jedi:`` Block
+------------------------------------
+
+execution:
+^^^^^^^^^^
+
+See :ref:`here <execution_yaml>` for details.
+
+configuration_file:
+^^^^^^^^^^^^^^^^^^^
+Supports ``base_file:`` and ``update_values:`` blocks (see the :ref:`updating_values` for details).
+
+files_to_copy:
+^^^^^^^^^^^^^^
+
+See :ref:`this page <files_yaml>` for details.
+
+.. code-block:: text
+
+   jedi:
+     files_to_copy:
+        f1: /path/to/f1
+
```

Comment:
Used the content from the `docs/shared/jedi.yaml`. Maybe a more real example would be useful here. I'm not sure what a real example would look like as of right now.

Reply:
I'm curious to hear what @christinaholtNOAA thinks, but I'm pretty ignorant about what real use cases look like. It seems like it could be a big topic, but maybe also something that's well-documented in JEDI itself, such that we could just redirect people to their documentation once we have real use cases and know more.